doc/source/serve/http-guide.md

```diff
@@ -1,12 +1,37 @@
-# Calling Deployments via HTTP
+# HTTP with Serve
 
 
+
+This section should help you:
+
+- Understand how to configure HTTP server location
+- Understand how to send HTTP request to serve deployment
+- Understand how Ray Serve integrate with FastAPI
+- Understand how to use customized HTTP Adapter
+
+
+## Configuring HTTP Server Locations
+
+By default, Ray Serve starts a single HTTP server on the head node of the Ray cluster.
+You can configure this behavior using the `http_options={"location": ...}` flag
+in {mod}`serve.start <ray.serve.start>`:
```

Comment:
@sihanwang41 we shouldn't document `serve.start` here (in favor of `serve.run` now)

Reply:
ping on this

Reply:
updated @edoakes